dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/data/file/RumFileWriter.kt

```diff
@@ -32,18 +39,44 @@ internal class RumFileWriter(
 
     override fun writeData(data: ByteArray, model: RumEvent): Boolean {
         val writeDataSuccess = super.writeData(data, model)
-        if (writeDataSuccess && model.event is ViewEvent) {
-            if (!lastViewEventFile.exists()) {
-                lastViewEventFile.createNewFile()
+        if (writeDataSuccess) {
+            val event = model.event
+            when (event) {
+                is ViewEvent -> persistViewEvent(data)
```

Comment:
Correct me if I am wrong but this PR seems incomplete to me, I see that we are not sending yet the `ResourceDropped`,`ActionDropped`, `ErrorDropped`,`LongTaskDropped` events.